### PR TITLE
wrap demo server strings for translation

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

* Wraps demo server strings so we can translate
* Adds a drop-shadow to banner

### Reviewer guidance

Please check over the strings. I had to reword them slightly to account for linking and parameterization of the username/password

### References

…

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

**Review by rtibbles — APPROVED** [review]

File: kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue [hunk 1]
```diff
@@ -72,6 +68,19 @@
...
+    $trs: {
+      demoServerHeader: 'Welcome to the Kolibri demo site!',
+      demoServerP1: 'You are welcome to explore with any of the three primary user types:',
+      demoServerL1: "Learner (username: '{user}', password: '{pass}') or access as a guest",
+      demoServerL2: "Coach (username: '{user}', password: '{pass}')",
+      demoServerL3: "Admin (username: '{user}', password: '{pass}')",
+      demoServerP2:
+        'This online demo site includes the features and functionalities of the latest version, but only holds a sample of the content that has been made available for offline use in Kolibri. It is intended for demonstration purposes and any user data on here may be cleared periodically.',
+      demoServerP3:
+        'To learn more about using Kolibri, take a look at our and better understand the platform in an offline context:',
```
Comment (radinamatic) [inline kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue]: Am I missing something here...? `...take a look at our and better...`

Comment (indirectlylit) [reply kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue]: thanks good catch!  slipped by @rtibbles ;-)

Comment (rtibbles) [reply kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue]: Only human...

Comment (indirectlylit) [reply kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue]: you claim

Comment (radinamatic) [reply kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue]: :smiling_imp:

**Pull request merged**